core/templates/dev/head/pages/exploration_editor/history_tab/HistoryServicesSpec.js

```diff
@@ -222,9 +224,9 @@ describe('Compare versions service', function() {
               }
             };
           });
-        statesData.states[stateName] = newStateData;
+        data.exploration.states[stateName] = newStateData;
       }
-      return statesData;
+      return data;
```

Comment:
Rather than having an awkward ``data`` variable, you could keep the ``statesData`` variable and return a new object inline:

```javascript
return {
  exploration: {
    states: statesData
  }
};
```